src/modules/lesson/services/lesson-hashtag.service.ts

```diff
@@ -11,69 +12,67 @@ export class LessonHashtagService {
   ) {}
 
   async createManyHashtag(
-    hashtags: string[],
+    lessonhashtagIds: number[],
     lessonId: number,
-  ): Promise<LessonHashtagEntity[]> {
-    const lessonIds = Array.from({ length: hashtags.length }, () => lessonId);
-    const lessonHashtags = this.dataStructureHelper.createManyMapper<
-      Pick<LessonHashtagEntity, 'lessonId' | 'tag'>
+  ): Promise<ReadLessonHashtagDto[]> {
+    const lessonIds = Array.from(
+      { length: lessonhashtagIds.length },
+      () => lessonId,
+    );
```

Comment:
제가 Array from 을 많이 안써봐서 그런데 이 부분을 꼭 Array from 으로 구현해야하는건가요??

Reply:
```suggestion
    const lessonIds = new Array(lessonhashtagIds.length).fill(lessonId);
```
코드에 문제는 없게지만은 보다 좋은 가독성을 위해 'fill' 메서드를 사용하는건 어떨까요?

Reply:
Array.fill이 있었군요! fill로 변경하였습니다